### PR TITLE
v0.2.0

### PR DESCRIPTION
**Pull request by Paebbels**
# New Features

* Added *main goals*, *features* and *example* code to README.
* Added a basic documentation website.
  * Navigation bar
  * Landing page
  * Installation instructions
  * Dependencies
  * Licenses

# Changes

* Renamed from `pyEDAA.Configure` to `pyEDAA.ToolSetup`.
* Changes due to an updated version of pyTooling/Actions.
  * Added more artifact names to the cleanup rule.
  
# Bug Fixes

* Add missing `doc/prolog.inc`.

--------------------
**Related PR:**
* https://github.com/edaa-org/pyEDAA.ToolSetup/pull/4

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+ROOT = Path(__file__).resolve().parent
+
+sys_path.insert(0, abspath('.'))
+sys_path.insert(0, abspath('..'))
+
+
+# ==============================================================================
+# Project information
+# ==============================================================================
+project =   "pyEDAA.CLITool"
```
Comment (Paebbels) [inline doc/conf.py]: It's not using `pyTooling.Packaging`.
Need to be changed in `pyEDAA.CLITool` too.

File: doc/conf.py [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+copyright = "2021-2021 Patrick Lehmann - Boetzingen, Germany"
```
Comment (Paebbels) [inline doc/conf.py]: Check copyright time span.

File: doc/index.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+   |SHIELD:png:ToolSetup-github| |SHIELD:png:ToolSetup-ghp-doc| |SHIELD:png:ToolSetup-gitter|
+   |SHIELD:png:ToolSetup-gha-test|
+
+.. Disabled shields: |SHIELD:png:ToolSetup-src-license| |SHIELD:png:ToolSetup-doc-license| |SHIELD:png:ToolSetup-pypi-tag| |SHIELD:png:ToolSetup-pypi-status| |SHIELD:png:ToolSetup-pypi-python| |SHIELD:png:ToolSetup-lib-status| |SHIELD:png:ToolSetup-codacy-quality| |SHIELD:png:ToolSetup-codacy-coverage| |SHIELD:png:ToolSetup-codecov-coverage| |SHIELD:png:ToolSetup-lib-dep| |SHIELD:png:ToolSetup-req-status| |SHIELD:png:ToolSetup-lib-rank|
+
+The pyEDAA.ToolSetup Documentation
+##################################
+
+*TBD*
```
Comment (Paebbels) [inline doc/index.rst]: Add main goals and features from README.

File: doc/index.rst [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+* `Patrick Lehmann <https://GitHub.com/Paebbels>`__ (Maintainer)
+* `Martin Zabel <https://GitHub.com/https://github.com/mzabeltud>`__
+* `Unai Martinez-Corral <https://GitHub.com/umarcor>`__
+* `and more... <https://GitHub.com/edaa-org/pyEDAA.ToolSetup/graphs/contributors>`__
+
+.. toctree::
+   :hidden:
+
+   Used as a layer of EDA² ➚ <https://edaa-org.github.io/>
+
```
Comment (Paebbels) [inline doc/index.rst:105]: Add sections for:
* Abstract information
* Abstract datamodel
* YAML based configuration
* Supported tools

File: doc/requirements.txt [hunk 5]
```diff
@@ -0,0 +1 @@
+sphinx
```
Comment (Paebbels) [inline doc/requirements.txt]: Check documentation requirements.

File: doc/index.rst (hunk 4 above)
Comment (Paebbels) [inline doc/index.rst:105]: Add dependencies.

File: doc/index.rst [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+.. raw:: latex
+
+   \part{Appendix}
+
+.. toctree::
+   :caption: Appendix
+   :hidden:
+
+   genindex
```
Comment (Paebbels) [inline doc/index.rst:149]: Add these sections:
* Link to coverage
* Link to static type check
* Documentation license

**Review by Paebbels — COMMENTED** [review]
All tasks are for @Paebbels.

File: doc/conf.py (hunk 1 above)
Comment (Paebbels) [reply doc/conf.py]: Done (for both).

File: doc/conf.py (hunk 2 above)
Comment (Paebbels) [reply doc/conf.py]: Now automated.

File: doc/index.rst (hunk 3 above)
Comment (Paebbels) [reply doc/index.rst]: Done.

File: doc/requirements.txt (hunk 5 above)
Comment (Paebbels) [reply doc/requirements.txt]: Done.

File: doc/index.rst (hunk 4 above)
Comment (Paebbels) [reply doc/index.rst:105]: Done.

File: doc/index.rst (hunk 6 above)
Comment (Paebbels) [reply doc/index.rst:149]: Done.

File: doc/index.rst (hunk 4 above)
Comment (Paebbels) [reply doc/index.rst:105]: Added headlines and empty pages.

**Comment by Paebbels** [comment]
@umarcor this PR will finalize the rename and includes a basic documentation.

Documentation text will follow in a new PR. I would like to finalize the rename with this PR.

**Comment by Paebbels** [comment]
@umarcor I enabled the blocked Shields to get e.g. Codacy. But it looks strange.

I thought these are just to enable shields and to update the Token in the Codacy URLs.  
Looks like there is a similar question then for the `index.rst` in Sphinx.

**Comment by Paebbels** [comment]
OK, I think I fixed the shields. I took them from ProjectModel and modified the URLs/Texts/Tokens.

**Review by umarcor — APPROVED** [review]
LGTM!

**Pull request merged**